src/main/java/io/pravega/perf/WriterWorker.java

```diff
@@ -107,7 +107,12 @@ private Performance createBenchmark() {
 
     @Override
     public Void call() throws InterruptedException, ExecutionException, IOException {
-        perf.benchmark();
+        try {
+            perf.benchmark();
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Again, why print if it's just going to be rethrown?

Reply:
only for debugging purpose.